config/app.php

```diff
@@ -232,6 +232,11 @@
         App\Providers\DatagridRendererProvider::class,
         App\Providers\PermissionsServiceProvider::class,
         App\Providers\EntitySetupServiceProvider::class,
+        /*
```

Comment:
The comment is somewhat useless, as the namespace of the class gives us a pretty strong hint